nilearn/datasets/atlas.py

```diff
@@ -429,6 +429,139 @@ def fetch_atlas_harvard_oxford(atlas_name, data_dir=None,
+
+        - "maps": nibabel.Nifti1Image, 4D maps if a probabilistic atlas is
+          requested and 3D labels if a maximum probabilistic atlas was
+          requested.
+
+        - "labels": string list, labels of the regions in the atlas.
+
+    """
+    atlas_items = ("maxprob-thr0-1mm", "maxprob-thr0-2mm",
+                   "maxprob-thr25-1mm", "maxprob-thr25-2mm",
+                   "maxprob-thr50-1mm", "maxprob-thr50-2mm",
+                   "prob-1mm", "prob-2mm")
+    if atlas_name not in atlas_items:
+        raise ValueError("Invalid atlas name: {0}. Please chose an atlas "
+                         "among:\n{1}".format(
+                             atlas_name, '\n'.join(atlas_items)))
+
+    url = "https://cdn.filesend.jp/private/1McfSSXL0-f67vqFrnEIbQZbfvS7YHn8ZTHoDL657Goj2FZFMnbsOp21SE1QpdYO/Juelich.tgz"
+
+    # For practical reasons, we mimic the FSL data directory here.
+    dataset_name = 'fsl'
+    data_dir = _get_dataset_dir(dataset_name, data_dir=data_dir,
+                                verbose=verbose)
+
+    opts = {'uncompress': True}
+    root = os.path.join('data', 'atlases')
+
+    label_file = "Juelich.xml"
+    label_file = os.path.join(root, label_file)
+
+    atlas_file = os.path.join(root, 'Juelich',
+                              'Juelich-' + atlas_name + '.nii.gz')
+    
+    atlas_img, label_file = _fetch_files(
+        data_dir,
+        [(atlas_file, url, opts), (label_file, url, opts)],
+        resume=resume, verbose=verbose)
+
+    names = {}
+    from xml.etree import ElementTree
+    names[0] = 'Background'
+    for label in ElementTree.parse(label_file).findall('.//label'):
+        names[int(label.get('index')) + 1] = label.text
+    names = list(names.values())
+
+    if not symmetric_split:
+        return Bunch(maps=atlas_img, labels=names)
+
+    atlas_img = check_niimg(atlas_img)
+
+    atlas = get_data(atlas_img)
+
+    labels = np.unique(atlas)
+    # Build a mask of both halves of the brain
+    middle_ind = (atlas.shape[0] - 1) // 2
+    # Put zeros on the median plane
+    atlas[middle_ind, ...] = 0
+    # Split every zone crossing the median plane into two parts.
+    left_atlas = atlas.copy()
+    left_atlas[middle_ind:, ...] = 0
+    right_atlas = atlas.copy()
+    right_atlas[:middle_ind, ...] = 0
+
+    new_label = 0
+    new_atlas = atlas.copy()
+    # Assumes that the background label is zero.
+    new_names = [names[0]]
+
+    for label, name in zip(labels[1:], names[1:]):
+        new_label += 1
+        left_elements = (left_atlas == label).sum()
+        right_elements = (right_atlas == label).sum()
+        n_elements = float(left_elements + right_elements)
+        if (left_elements / n_elements < 0.05 or
+                right_elements / n_elements < 0.05):
+            new_atlas[atlas == label] = new_label
+            new_names.append(name)
+            continue
+        new_atlas[right_atlas == label] = new_label
+        new_names.append(name + ', left part')
```

Comment:
I should check again cause i took this part from the fetch_harvard_oxford function

Reply:
OK :+1: 
Again, I didn't look extensively, so this might be a stupid comment...

Reply:
But do we actually need all that ?

Reply:
Do you mean that the Juelich atlas has the same structure as Harvard-Oxford ?

Reply:
as @NicolasGensollen said  right and left are swapped. the final result is correct because this turns out to compensate another bug (the orientation of the x axis ie the sign of the first entry of the affine, is ignored, see above)

Reply:
> But do we actually need all that ?

I agree with @bthirion : can't we do without all this splitting and merging stuff and just provide the standard versions?